atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -1125,14 +1122,12 @@ private void putWritesAndLogIfTooLarge(Map<Cell, byte[]> values, SortedMap<Cell,
                 long newVal = byteCount.addAndGet(toAdd);
                 if (newVal >= TransactionConstants.WARN_LEVEL_FOR_QUEUED_BYTES
                         && newVal - toAdd < TransactionConstants.WARN_LEVEL_FOR_QUEUED_BYTES) {
-                    log.warn("A single transaction has put quite a few bytes: {}", newVal, new RuntimeException(
-                            "This exception and stack trace are provided for debugging purposes."));
-                }
-                if (newVal >= TransactionConstants.ERROR_LEVEL_FOR_QUEUED_BYTES
-                        && newVal - toAdd < TransactionConstants.ERROR_LEVEL_FOR_QUEUED_BYTES) {
-                    log.warn("A single transaction has put too many bytes: {}. This can potentially cause"
-                            + " out-of-memory errors.", newVal, new RuntimeException(
-                            "This exception and stack trace are provided for debugging purposes."));
+                    log.warn("A single transaction has put quite a few bytes: {}. "
+                            + " Enable debug logging for more information", newVal);
```

Comment:
As above, note that we'll have 2 spaces between sentences (while the current message has 1).